src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -524,6 +524,8 @@ public void parse_configOrLocationsSimilar_success() throws Exception {
                 .getRunConfiguration(locationCliArguments)
                 .getRepoConfigurations();
 
+        System.out.println(actualRepoConfigs.get(0));
```

Comment:
Did you intend to leave this in?

Reply:
Oh no, I must have forgotten to remove it after debugging. My bad! I will remove it in another commit!